src/target/puya.c

```diff
@@ -0,0 +1,171 @@
+#include "general.h"
+#include "target.h"
+#include "target_internal.h"
+#include "buffer_utils.h"
+#include "cortexm.h"
+#include "adiv5.h"
+
+/* Flash */
+#define FLASH_START     0x08000000U
```

Comment:
These definitions need to get the same suffixing treatment (`PUYA_`) as the ones from your other PR did